R/fit.R

```diff
@@ -71,22 +71,47 @@ CmdStanFit <- R6::R6Class(
       summary
     },
 
+    # print summary table without using tibbles
     print = function(variables = NULL, ..., digits = 2, max_rows = 10) {
-      # print summary table without using tibbles
-      out <- self$summary(variables, ...)
+      # FIXME: treat sample and other methods the same once we have
+      # self$metadata() (or equivalent) for all methods
+      if (self$runset$method() == "sample") {
+        all_variables <- self$sampling_info()$model_params
+      } else {
+        all_variables <- posterior::variables(self$draws())
+      }
```

Comment:
@rok-cesnovar related to our discussion at https://github.com/stan-dev/cmdstanr/pull/218#discussion_r444982784, only the sample method has `sampling_info()`. Once we have the same thing for all methods this can be changed to use that. Until then for the other methods we need to get the variable names from `draws()`.